ramalama/cli.py

```diff
@@ -721,6 +721,15 @@ def run_parser(subparsers):
     parser.add_argument(
         "ARGS", nargs="*", help="Overrides the default prompt, and the output is returned without entering the chatbot"
     )
+    # Disable network access by default, and give the option to pass any supported network mode into
+    # podman if needed:
+    # https://docs.podman.io/en/latest/markdown/podman-run.1.html#network-mode-net
+    parser.add_argument(
+        "--network-mode",
+        type=str,
+        default="none",
```

Comment:
@rhatdan should we use private by default?

Reply:
No none is a better default. I don't see a container needing access to the network for `run`, might need `private` for serve.